packages/database/scripts/dumpDatabase.sh

```diff
@@ -55,7 +75,11 @@ target_path="$(
     cd "$target_dir"
     pwd
 )/$DUMP_FILE_NAME"
+target_zip_path="$target_path.gz"
+
+show_loading_spinner "Dumping database to $target_zip_path" "PGPASSWORD=$DB_PG_PASSWORD pg_dump \"host=$host user=postgres dbname=tupaia sslmode=require sslkey=$identity_file\" -Z1 -f $target_zip_path"
```

Comment:
Adding the `-Z1` argument turns on compression of the dump file. I skipped doing this previously as it didn't really seem to speed up the process at all.

However for the folks at home saving 8GB of data download each time you dump the database is probably worth it